psi4/share/psi4/fsapt/fsaptdiff.py

```diff
@@ -57,62 +57,62 @@
 
     fh = open('%s/fsapt.dat' % dirA, 'w')
     fh, sys.stdout = sys.stdout, fh
-    print '  ==> F-ISAPT: Links by Charge <==\n'
+    print('  ==> F-ISAPT: Links by Charge <==\n')
     stuffA = computeFsapt(dirA, False)
-    print '   => Full Analysis <=\n'
+    print('   => Full Analysis <=\n')
     printOrder2(stuffA['order2'], stuffA['fragkeys']) 
-    print '   => Reduced Analysis <=\n'
+    print('   => Reduced Analysis <=\n')
     printOrder2(stuffA['order2r'], stuffA['fragkeysr']) 
```

Comment:
@amjames that should go away too.  There's a keyword to find_package, EXACT, that requires it to find a library/executable with exactly that version.  I simply take the version from the first call (now find PythonLibs because I took this solution from Pulsar where we looked for that first) and force the second find_package (the interpreter) to find that version.